src/ResourceManager/KeyVault/Commands.KeyVault/Properties/AssemblyInfo.cs

```diff
@@ -29,8 +29,8 @@
 [assembly: CLSCompliant(false)]
 
 [assembly: Guid("2994548F-69B9-4DC2-8D19-52CC0C0C85BC")]
-[assembly: AssemblyVersion("1.1.3.1")]
-[assembly: AssemblyFileVersion("1.1.3.1")]
+[assembly: AssemblyVersion("2.3.0")]
+[assembly: AssemblyFileVersion("2.3.0")]
```

Comment:
why do we need this change. We are looking for minimal changes to bring ObjectId as string